mongo_handler.py

```diff
@@ -169,24 +174,23 @@ def dump_vulnerabilities(collection):
                                                             false_negative))
 
 
-# Static variable to keep vulnerabilities map in memory
-vulnerabilities_map = {}
-
-
-def get_oracle(vulnerability_id, version):
```

Comment:
Please keep the old code for the vulnerabilities already mapped. If the file '..._vulnerability.txt'  is not found, than do your check.

Reply:
There you go! 9af1229